src/main/java/com/odysseusinc/athena/api/v1/controller/converter/vocabulary/StringToVocabularyVersionDTOConverter.java

```diff
@@ -0,0 +1,31 @@
+package com.odysseusinc.athena.api.v1.controller.converter.vocabulary;
+
+import com.odysseusinc.athena.api.v1.controller.dto.vocabulary.VocabularyVersionDTO;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.core.convert.support.GenericConversionService;
+
+public class StringToVocabularyVersionDTOConverter implements Converter<String, VocabularyVersionDTO>, InitializingBean {
```

Comment:
Converter should appear in application context in any way,
e.g. with `@Component` annotation